<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogInversionParaSetting</class>
 <widget class="QDialog" name="DialogInversionParaSetting">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>364</width>
    <height>608</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>364</width>
    <height>608</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>364</width>
    <height>608</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>反演参数设置</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">QGroupBox 
{border: 2px solid #0000EE;
 border-radius: 5px;
margin-top: 1ex;
}

QGroupBox::title {
subcontrol-origin: margin;
subcontrol-position: top center;
padding: 0 3px;
}</string>
     </property>
     <property name="title">
      <string>参数</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,2">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>反演方法选择开关：      </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBox_selectionSwitch">
          <item>
           <property name="text">
            <string>最小二乘</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>GA</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>联合</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>蚁群</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_14">
        <item>
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>异常体编辑信息对比开关：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBox_compare">
          <property name="text">
           <string/>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="2" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_9" stretch="0,0">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>反演次数上限：          </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_INVMAX">
          <property name="text">
           <string>3</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="3" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,2">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>误差容许值：            </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_EPS">
          <property name="text">
           <string>0.000001</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="4" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="1,2">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>相对拟合差的收敛数据：  </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_EPSRMS">
          <property name="text">
           <string>0.000003</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="5" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,2">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>光滑度矩阵的计算方法：  </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBox_computingMethod">
          <item>
           <property name="text">
            <string>不含距离因子</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>含距离因子</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item row="6" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="1,2">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>反演模型中的电阻率上限：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_BoundU">
          <property name="text">
           <string>2000</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="7" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="1,2">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>反演模型中的电阻率下限：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_BoundL">
          <property name="text">
           <string>7</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="8" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_8" stretch="1,2">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>数据加权：              </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBox_dataWeighting">
          <item>
           <property name="text">
            <string>线性</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>指数</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>二次</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>无加权</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item row="9" column="0">
       <widget class="QGroupBox" name="groupBox_3">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>45</height>
         </size>
        </property>
        <property name="title">
         <string>阻尼因子</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_13" stretch="1,2">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_7">
           <item>
            <widget class="QRadioButton" name="radioButton_use">
             <property name="text">
              <string>使用</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="radioButton_notUse">
             <property name="text">
              <string>不使用</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_Lamda">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="minimumSize">
            <size>
             <width>178</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>178</width>
             <height>16777215</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="10" column="0">
       <widget class="QGroupBox" name="groupBox_2">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>103</height>
         </size>
        </property>
        <property name="title">
         <string>PCG参数设置</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout_11" stretch="0,0">
           <property name="spacing">
            <number>0</number>
           </property>
           <item>
            <widget class="QLabel" name="label_PCGEPS">
             <property name="text">
              <string>PCG法解方程时的均方差：    </string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEdit_PCGEPS">
             <property name="text">
              <string>0.000003</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="1" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout_12" stretch="0,0">
           <property name="spacing">
            <number>0</number>
           </property>
           <item>
            <widget class="QLabel" name="label_11">
             <property name="text">
              <string>PCG收敛容许误差：          </string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEdit_PCGRELERR">
             <property name="text">
              <string>0.1</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="2" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout_10" stretch="0,0">
           <property name="spacing">
            <number>0</number>
           </property>
           <item>
            <widget class="QLabel" name="label_10">
             <property name="text">
              <string>PCG迭代次数上限：          </string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEdit_PCGITMAX">
             <property name="text">
              <string>800000</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>radioButton_use</sender>
   <signal>clicked()</signal>
   <receiver>DialogInversionParaSetting</receiver>
   <slot>onClickedUseLAMDA()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>53</x>
     <y>461</y>
    </hint>
    <hint type="destinationlabel">
     <x>67</x>
     <y>-6</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton_notUse</sender>
   <signal>clicked()</signal>
   <receiver>DialogInversionParaSetting</receiver>
   <slot>onClickedNotUseLAMDA()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>119</x>
     <y>459</y>
    </hint>
    <hint type="destinationlabel">
     <x>136</x>
     <y>4</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onClickedUseLAMDA()</slot>
  <slot>onClickedNotUseLAMDA()</slot>
 </slots>
</ui>
